packages/synthetics-sdk-api/src/handlers.ts

```diff
@@ -29,9 +29,14 @@ import { getInstrumentedLogger } from './auto_instrumentation';
 instantiateMetadata();
 
 const asyncFilenamePrefix = 'async ';
+const syntheticExecutionIdHeader = 'Synthetic-Execution-Id';
 const runSynthetic = async (
   // eslint-disable-next-line  @typescript-eslint/no-explicit-any
-  syntheticCode: (args: { logger: Logger }) => any
+  syntheticCode: (args: {
+    logger: Logger;
+    executionId: string | undefined;
+  }) => any,
```

Comment:
Since this is now on multiple lines, the eslint annotation is no longer working. Can you fix this? (should be a way to make this multi line)

Reply:
I didn't see documentation for disabling multiple lines ([src](https://eslint.org/docs/latest/use/configure/rules)), so I moved the comment to another line